apps/web/src/ui/swap/cross-chain/cross-chain-swap-trade-review-dialog.tsx

```diff
@@ -89,6 +92,28 @@ import {
   useDerivedStateCrossChainSwap,
 } from './derivedstate-cross-chain-swap-provider'
 
+function getConfig(trade: UseCrossChainTradeReturn | undefined) {
+  if (!trade) return {}
+
+  if (trade.functionName === SushiXSwapFunctionName.Bridge) {
+    return {
+      abi: sushiXSwap2Abi_swap,
+      functionName: 'swap',
+      args: trade.writeArgs as NonNullable<SushiXSwapWriteArgsBridge>,
+      value: BigInt(trade.value ?? 0) as any,
+    } as const
+  }
+
+  if (trade.functionName === SushiXSwapFunctionName.SwapAndBridge) {
+    return {
+      abi: sushiXSwap2Abi_swapAndBridge,
+      functionName: 'swapAndBridge',
+      args: trade.writeArgs as NonNullable<SushiXSwapWriteArgsSwapAndBridge>,
+      value: BigInt(trade.value ?? 0) as any,
```

Comment:
cant these br undefined?

Reply:
don't think so. as long as trade is defined, writeArgs is, too

Reply:
I mean the value: undefined, and the as any

https://wagmi.sh/core/api/actions/call#value